examples/lessons/10/app.js

```diff
@@ -5,7 +5,7 @@ import {
 
 import {
   Matrix4
-} from 'math.gl'
+} from 'math.gl';
```

Comment:
Nit: Let's keep this as a one line import. We want our examples to be fairly compact. The multiline style is suitable when importing lots of symbols from a module.
```
import {Matrix4} from 'math.gl'
```